### ISSUE TITLE
Error: Cannot find module './createHandler' from 'index.cjs.js'

### ISSUE THREAD

**Issue by yurist38** [bumble-org/jest-chrome#1]
**Describe the bug**
I just got the package updated from v0.5.1 to v0.5.2 and I see this error now:

```
Cannot find module './createHandler' from 'index.cjs.js'
```

**To Reproduce**
1. Clone [this repo](https://github.com/yurist38/linkedin-video-downloader)
2. Checkout to the branch `issue-jest-chrome`
3. Run tests `npm t`

**Expected behavior**
All tests must pass without any errors and coverage 100%.

**Desktop (please complete the following information):**
 - OS: MacOS Catalina 10.15.4
 - Node.js: 12.16.1

**Additional context**

**Comment by AmySteam** [comment]
Hi Yuri! 
Thanks for bringing this to our attention. We have a few deadlines this week, but we will give this top priority on Friday.
All the best,
Amy

**Comment by yurist38** [comment]
Sure, I just wanted to report it to you that you're aware now.
I also wanted to thank you guys for doing this great job of bringing nice tools for extension developers. I didn't see them before but my next chrome extension I'll build with your stack I think (even despite the fact that I'm not a big fan of RX 😄). Have you ever thought of making some framework out if it? It would be great! I'd love to contribute...

**Comment by wosephjeber** [comment]
I just encountered this issue as well. I can confirm that downgrading to `0.5.1` works for now.

And I second the 👍 for the work on this repo. I recently entered the world of Chrome extension development, and it's nice to see some really helpful tooling being developed by Bumble.

**Comment by jacksteamdev** [comment]
@yurist38 @wosephjeber Thanks for all the support! 😁

I've fixed this now, and added some tests for the build files so this won't happen again.

@yurist38 I agree, `rxjs` integration should be optional. I've removed `rxjs` from this project, and I'm going to release the other libs without it in the near future.

@wosephjeber Actually, we're not associated with Bumble... 😅

We chose that name before we found out about the app.  We are migrating these packages soon to a new organization to help avoid any brand confusion.

The new org name will be [`extend-chrome`](https://github.com/extend-chrome) and we're excited to see people using these tools!
